playbooks/installation/installation.adoc

```diff
@@ -1,280 +1,79 @@
+The OpenShift Container Platform Smart Start is a reference architecture for a Highly Available installation of OpenShift. This document outlines the architecture of such an installation and walks through the installation process.
 
+== Architecture
 
-=== DNS
+image::/images/ocp_smart_start_diagram.png[Smart Start Architecture]
 
-If the organization has the ability to add an A record to the corporate DNS, supporting wildcard name resolution to the OpenShift environment, that DNS server should be utilized.
+The diagram above depicts the architecture of the Smart Start environment. It consists of 3 Masters and 5 Nodes. We further subdivide the nodes into two groups. Two of them get designated to run OpenShift's router and image registry. We refer to these as _Infrastructure Nodes_. The other three will run the actual application workloads. We call these _Application Nodes_ or _Compute Nodes_.
 
-If adding those entries is problematic, a quick DNS solution for OpenShift is to use dnsmasq.
+NOTE: For more information on these concepts and components, see our link:/playbooks/fundamentals/building_blocks_openshift{outfilesuffix}[Building Blocks of OpenShift] doc.
 
-=== Node Sizing
+== Before You Begin
 
-The Offcial Documentation lists some link:https://docs.openshift.com/enterprise/3.0/admin_guide/install/prerequisites.html#system-requirements[minimal system requirements] for OpenShift masters and nodes. Beyond that, there are several things to consider when deciding on the sizing for your nodes.
+=== Provision Servers
 
-==== Number of Nodes & Overall Environment Size
+The Smart Start architecture requires the following Servers(VMs) be provisioned:
 
-When deciding how to size nodes, it's helpful to have an idea of the final size of your environment. This is important because knowing the total amount of resources your workloads will consume will help to dictate the amount of capacity loss your environment can handle. This is especially relevant in smaller environments, where the loss of one node could have significant impact on the performance and capacity of your overall environment. It's a good rule of thumb for environment sizing is to take the max expected total workload size and add 30% to that. This leaves about 10% for overhead on your nodes, plus 20% extra to account for node failure.
+* 3 _Masters_
+  ** RHEL 7.2 minimal installation
+  ** 8 GB Memory
+  ** 2 Cores
+  ** 30 GB Root Drive
+  ** An additional 5 GB disk or partition mounted at /var/lib/etcd
+* 2 _Infrastructure Nodes_
+  ** RHEL 7.2 minimal installation
+  ** 8 GB Memory
+  ** 4 Cores
+  ** 30 GB Root Drive
+  ** An additional 20 GB disk or partition mounted at /var/lib/origin
+  ** An additional 50 GB disk, unformatted and unmounted
+* 3 _Application Nodes_
+  ** 16 GB Memory
+  ** 4 Cores
+  ** 30 GB Root Drive
+  ** An additional 20 GB disk or partition mounted at /var/lib/origin
+  ** An additional 50 GB disk, unformatted and unmounted
```

Comment:
Optional file system for OpenShift volumes `/var/lib/origin` as they can grow due to storage from ephemeral containers